src/entries/nesting-selectors.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: layouts/entry/index.njk
+title: Nesting Selectors
+date: 2021-02-19
+tags:
+  - scss
+nextEntryTitle: Mixins
+---
+
+One of the most obvious changes SCSS brings to writing styles is the ability to nest them. In CSS, selectors can only contain rules. In SCSS, selectors can contain other selectors, which is called nesting.
+
+<div class="two-up-code">
+
+```html
+<ul class="list">
+  <li>
+    item
+  </li>
+  ...
+  <li>
+    item
+  </li>
+</ul>
+```
+
+```scss
+.list {
+  li {
+    margin-bottom: 0.5em;
+  }
+}
+```
+
+</div>
+
+At the most basic level, nesting rules is a shorthand for adding combinators ([read more](/entries/what-is-css/#descendant-combinator%3A-%5Bspace%5D)) to multiple selectors at once, without having to rewrite the first ingredient. In a more abstract sense, nesting rules allows us to declaratively match the structure of styles to the structure of markup. It also helps reduce repetition when writing code, enables some clever options for composition, and when used smartly, can make styles highly readable and structured.
+
+## Nesting & Specificity
+
+Before we get into it, let's talk about what not to do. It's tempting to nest deeply, and match styles to markup exactly. Doing so can get you into trouble very quickly, as every additional selector adds specificity to a rule. In general, when writing styles (SCSS and CSS), your goal should be to **maintain the minimum level of specificity at all times.**
+
+Here's an inadvisable example of nesting:
```

Comment:
Maybe this is tacky, but maybe we could include a like 👎 or 🚫 or 🙅 or something to reinforce visually that this is bad practice? Totally just a suggestion!